ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/compliance.coverage.utils.ts

```diff
@@ -1,10 +1,24 @@
-import { ProgressVariant } from '@patternfly/react-core';
-
 import {
     ComplianceCheckStatus,
     ComplianceCheckStatusCount,
 } from 'services/ComplianceEnhancedService';
 
+import { LabelProps } from '@patternfly/react-core';
```

Comment:
Somehow in deleting the old and adding the new, this `import` ended up below the other.